### PR TITLE
Add gdspy

### PR DESCRIPTION
**Pull request by willsALMANJ**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by willsALMANJ** [comment]
For what it's worth, I tested that the test suite passed with Python 3.4 / numpy 1.14 and Python 3.8 / numpy 1.19.

**Comment by willsALMANJ** [comment]
@conda-forge/help-python-c I tested that the Windows and Linux Python 3.8 build artifacts work for me.

File: recipes/gdspy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - gdspy
+
+about:
+  home: https://github.com/heitzmann/gdspy
+  license: BSL-1.0
+  license_file: LICENSE.txt
+  summary: 'A Python module for creation and manipulation of GDSII stream files'
+
+  # The remaining entries in this section are optional, but recommended.
```
Comment (scopatz) [inline recipes/gdspy/meta.yaml:36]: please remove comments

**Comment by willsALMANJ** [comment]
Thanks, @scopatz! By the way, I know that some in line comments have meaning in `meta.yaml` but are there any leading comments that do (meaning nothing but whitespace in front of the `#`)? I wonder if the linter could flag those comments automatically.

**Pull request merged**